vespa-http-client/src/test/java/com/yahoo/vespa/http/client/FeedClientTest.java

```diff
@@ -55,6 +55,7 @@ public void testGetStatsAsJson() throws Exception {
         feedClient.stream(DOCID, "blob");
         while (resultsReceived.get() == 0) {Thread.sleep(3); }
         String stats = feedClient.getStatsAsJson();
+        System.out.println("Stats: "+ stats);
```

Comment:
Leftover from debugging or added to make it easier to verify output of test?

Reply:
Removed, thanks.